keps/sig-node/4188-kubelet-pod-readiness-api/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: New kubelet gRPC API with endpoint returning local pods information
+kep-number: 4188
+authors:
+  - "@kl52752"
+  - "@robscott"
+owning-sig: sig-node
+participating-sigs: []
+status: implementable
+creation-date: 2023-09-05
+reviewers:
+approvers:
+  - "@sig-node-tech-leads"
+see-also:
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.29"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.29"
+  beta: "v1.30"
+  stable: "v1.31"
```

Comment:
this seems a very aggressive schedule, but can be reviewed later and adjusted if needed

Reply:
In general it's a good practice to give some time (like a couple of release cycles) to folks to consume and integrate the change, so bugs or issues or rough edges surface. But we can surely review later and discuss during the KEP graduation process.